src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java

```diff
@@ -112,6 +115,25 @@ public ResponseEntity<Page<AdvertisementTitleDto>> getPageOfAdvertisementsByKeyw
                 new ResponseEntity<>(allByKeyword, HttpStatus.OK);
     }
 
+    @GetMapping(ApiKey.ADV_SEARCH_PAGINATED_BY_CATEGORY_ID)
+    @Operation(summary = "Find advertisements by category id")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK"),
+            @ApiResponse(responseCode = "400", description = "BAD REQUEST"),
+            @ApiResponse(responseCode = "404", description = "NOT FOUND")})
+    public Page<AdvertisementTitleDto> getPageOfAdvertisementsByCategoryId(
+            @PathVariable("category_id") Long id,
```

Comment:
Why don't you use **categoryId** instead of **id** in the variable name?

Reply:
+

Reply:
use caregoryId